content/news/XXXX-XX-XX-build-infrastructure-updates.md

```diff
@@ -0,0 +1,79 @@
+
+## Developer Experience
+
+All of this work has been very tedious. We would have much rather spent this effort writing fun new features for Mixxx, but this had to be done to keep the project going. Now that it is working, developing for Mixxx is much nicer.
+
+As we got the new setup going with GitHub Actions, we wrote scripts that can be used both by GitHub Actions and by developers that want to set up a development environment on their computer. The scripts automatically download the archive of Mixxx's dependencies. On macOS, the script sets the appropriate environment variables. On Windows, the script generates a `CMakeSettings.json` file which can be used to setup Visual Studio to work on Mixxx very easily. We hope this makes Mixxx more welcoming for Windows and macOS developers to contribute.
+
+Developers on every OS can now easily have their code built and tested automatically on Windows, macOS, and Linux simply by pushing commits to GitHub, and we can ask users to give feedback using the build artifacts from GitHub Actions.
+
+On macOS and Linux, the builds on GitHub Actions only take a few minutes now that we are using [ccache](https://ccache.dev/) which was [easy to implement with CMake](https://cmake.org/cmake/help/latest/prop_tgt/RULE_LAUNCH_COMPILE.html). We have tried getting compiler caching working on Windows with [sccache](https://github.com/mixxxdj/mixxx/pull/3618) and [clcache](https://github.com/mixxxdj/mixxx/pull/3473) but have not succeeded yet, so the Windows builds still take about 40 minutes.
+
+Historically, issues with the build servers have been a major factor why Mixxx releases have been so few and far between. We hope that by having this automated on a reliable service that we do not need to maintain ourselves, we can focus more on programming cool new features and getting Mixxx releases published more regularly.
+
+If you want to [get involved](https://mixxx.org/get-involved/) in making Mixxx more awesome, come introduce yourself on our [Zulip chat](https://mixxx.zulipchat.com/#narrow/stream/109123-introduce-yourself). This new infrastructure makes it easier both for developers to contribute code and for users to test new code, so you do not need to be a coder to contribute!
```

Comment:
The second half of the sentence does not follow from the first half, which makes it sound weird. 

```suggestion
If you want to [get involved](https://mixxx.org/get-involved/) in making Mixxx more awesome, come introduce yourself on our [Zulip chat](https://mixxx.zulipchat.com/#narrow/stream/109123-introduce-yourself). You do not need to be a coder to contribute - our new infrastructure makes contributing easier both for developers and testers!
```